uvicorn/main.py

```diff
@@ -36,8 +37,47 @@
 )
 
 
+def _setup_defaults_from_settings_script(
+    ctx: click.Context, param: click.Parameter, value: typing.Optional[str]
+):
+    if value is None:
+        return
+
+    members = runpy.run_path(value)  # type: dict
+
+    try:
+        config = members["config"]  # type: Config
```

Comment:
We have dropped Python 3.5 support. PEP 526 style annotations is more better : )

Reply:
So much happened in 7 months! 😁

Reply:
@Hanaasgi Actually it seems like there’s still a PyPy 3.5 step in our CI build…? I rebased from master, is there anything else I’m missing?

Reply:
:sweat_smile: Pending https://github.com/encode/uvicorn/pull/424 . Master branch CI is using pyp3.5.